src/iuse.cpp

```diff
@@ -1841,6 +1841,31 @@ int iuse::purify_iv(player *p, item *it, bool, const tripoint& )
     return it->type->charges_to_use();
 }
 
+void spawn_spores( player *p ) {
```

Comment:
You don't seem to be affecting the player here.
That could be a const reference instead, as the player isn't expected to be `nullptr` here.